### ISSUE TITLE
[enhancement] use rich Python library to prettify output of khal printics

### ISSUE THREAD

**Issue by tgy** [pimutils/khal#1280]
everything's in the title

i just used the `khal printics` command and i'm sure we can do better!

the [rich](https://github.com/Textualize/rich) library is pretty awesome and could be used for that purpose

**Comment by geier** [comment]
I agree, we totally can do better.

I had a look at rich and textual #1171 (which is from the same people and probably built on top of rich) some time ago and it looks pretty good. My issue at the moment is, that I would prefer if the ways khal and ikhal draw it's UI converges rather than diverges. Re-doing ikhal in textualize looks like a lot of work.

For the moment I would suggest not to spend too much work on doing this in rich, but let's discuss.

**Comment by tgy** [comment]
i agree this is not the number 1 priority, but opened the issue anyway so as not to forget about it :-)
